ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -382,8 +382,7 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
                         this.visitedSectionIndexes = this.visitedSectionIndexes
                             .map((value, index) => index <= this.currentSectionIndex);
                     }
-                }),
-                take(1)
```

Comment:
If we take it only once it does not update  `this.currentSectionIndex` (line 381) and uses a previous activity value.